### ISSUE TITLE
Multishell ACT HSVS recon workflow uses SS3T response functions

### ISSUE THREAD

**Issue by araikes** [PennLINC/qsirecon#137]
@mattcieslak 

In the process of testing things for all of the other issues over the last 24 hours, I noticed that `mrtrix_multishell_msmt_ACT-hsvs` uses `dhollander` for the response function estimation (using the `SS3TDwiResponse`) interface. 

```
	 [Node] Setting-up "qsirecon_0_0_wf.sub-001_mrtrix_multishell_msmt_hsvs.sub_001_ses_01_dir_AP_run_001_space_T1w_desc_preproc_recon_wf.msmt_csd.estimate_response" in "/output/scratch/qsirecon_0_0_wf/sub-001_mrtrix_multishell_msmt_hsvs/sub_001_ses_01_dir_AP_run_001_space_T1w_desc_preproc_recon_wf/msmt_csd/estimate_response".
241025-10:53:46,553 nipype.workflow INFO:
	 [Node] Executing "estimate_response" <qsirecon.interfaces.mrtrix.SS3TDwi2Response>
241025-10:55:00,857 nipype.workflow INFO:
```

Given that `msmt_5tt` is a response function option, is there a specific rationale for using `dhollander` in the HSVS spec?

**Comment by mattcieslak** [comment]
Luckily this one isn't a bug! When we added ss3t to the recon workflows, we learned that there was an updated `dhollander` method implemented in the ss3t fork. That method should be available in the mrtrix that is now included in the qsirecon container, so we should switch to explicitly use the mrtrix3 version unless ss3t is explicitly being requested.

**Comment by araikes** [comment]
Great. Just observed that and wasn't sure if it was intentional or a by product of other processes.

**Comment by tsalo** [comment]
Is this something we should document somewhere or is it good to close as is?

**Comment by araikes** [comment]
Depends... what's the current behavior?

**Comment by mattcieslak** [comment]
I don't think it is explicitly stated in the boilerplate that we use the updated `dhollander`. We should include that in the boilerplate IMO

**Comment by araikes** [comment]
Agreed. Equally important, though not outward facing, is whether the interface is accurately named.
